cmd/main.go

```diff
@@ -64,7 +71,16 @@ func main() {
 	opts.BindFlags(flag.CommandLine)
 	flag.Parse()
 
-	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
+	zapLogger := zap.New(zap.UseFlagOptions(&opts))
+
+	ctrl.SetLogger(zapLogger)
+	klog.SetLogger(zapLogger)
+
+	linodeToken, ok := os.LookupEnv("LINODE_API_TOKEN")
```

Comment:
optional: The standard environment variable we use for DX-owned integrations is `LINODE_TOKEN`, so it might make sense to also use that here. It might also make sense to move the `LINODE_API_TOKEN` string into a const.

Reply:
+1 for switching this to `LINODE_TOKEN`